LibreHardwareMonitorLib/Hardware/Motherboard/Lpc/EC/EmbeddedController.cs

```diff
@@ -158,7 +158,7 @@ public BoardInfo(Model model, BoardFamily family, params ECSensor[] sensors)
                 ECSensor.TempChipset, ECSensor.TempCPU, ECSensor.TempMB,
                 ECSensor.TempVrm, ECSensor.FanChipset, ECSensor.CurrCPU, ECSensor.VoltageCPU
             ),
-            new(new Model[] {Model.ROG_CROSSHAIR_VIII_HERO, Model.ROG_CROSSHAIR_VIII_HERO_WIFI }, BoardFamily.Amd500,
+            new(new Model[] {Model.ROG_CROSSHAIR_VIII_HERO, Model.ROG_CROSSHAIR_VIII_HERO_WIFI, Model.ROG_CROSSHAIR_VIII_FORMULA }, BoardFamily.Amd500,
```

Comment:
But Formula, unlike Hero, does not have water sensors, does it?

Reply:
If this is the correct model, it seems it does have water headers and connectors:
https://rog.asus.com/motherboards/rog-crosshair/rog-crosshair-viii-formula-model/spec